### ISSUE TITLE
(Hot Fix: Disable Integrated Graphics) Driver Version 4091 and 4123 Is Incompatible w/ Modern Warfare II

### ISSUE THREAD

**Issue by JohnnyTex** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#233]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Call of Duty: Modern Warfare II

### Game Platform [Required]

- [ ] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [X] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

i5 12600k

### Graphic Card [Required]

ARC A770 LE

### GPU Driver Version [Required]

- [X] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034
- [ ] 31.0.101.4032
- [ ] 31.0.101.3975
- [ ] 31.0.101.3959
- [ ] 31.0.101.3802
- [ ] 31.0.101.3793
- [ ] 31.0.101.3491
- [ ] 31.0.101.3490

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[24ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/10610070/24ssu.txt)

### Description and steps to reproduce [Required]

On the latest ARC driver, the game won't recognize my GPU and thus crash during the opening boot sequence. I can't even make it to the menus. What's odd was that I think at first I was able to do this, but one day it stopped working. I went to the Intel Discord to ask if others have had this issue, and it turns out yes, others are saying I have to go back to v.4090 to play this game. I went back and lo and behold, I can play the game again. 

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [ ] High
- [X] Ultra

### Game resolution [Required]

2560 x 1440

### Game VSync [Required]

Off

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

Basically the Ultra preset, with a few tweaks to things like reflections and textures as well as using AI upscaling.

**Comment by IntelSupport-Arun** [comment]
Hi @JohnnyTex ,
 
We have the following questions to you:
1. It worked before, but then suddenly there was an issue when you tried to launch it. Did you try changing any system settings or ARC control settings? 
2. Since the issue is not happening with v.4090 could you try a clean installation  of 31.0.101.4091 by selecting the below option
![Fresh_Installation_ARC_driver](https://user-images.githubusercontent.com/91881865/217195357-1697d6fa-ae79-4cd8-a4df-a5a9803741e3.jpg)
3. May I know  the issue is specific to ARC? did you get any chance to verify the issue with other brands?
4. Please confirm the version of MWII, is it vault or Standard?
![image](https://user-images.githubusercontent.com/91881865/217196473-246b828f-876b-4cd8-b719-19beedae140c.png)

**Comment by JohnnyTex** [comment]
1. Nothing big, and it only seemingly worked once before it ceased to work. I'll be real, I could definitely be misremembering that. 
2. I did the clean install of 4091 on my first install, and I had done so after a DDU as well. 
3. I don't have another GPU. 
4. It's the standard version; this is the version of MW2 that came bundled w/ ARC.

**Comment by JohnnyTex** [comment]
This issue has seemingly been resolved in Driver version 4123. I'll continue testing this to make sure it has been, and if so I'll close this case. I'd recommend rolling out a new stable driver including the fix (should it be true) swiftly though.

**Comment by IntelSupport-Arun** [comment]
Hi @JohnnyTex ,

Thank you for the result with 4021 driver. You may continue testing the latest driver and let me know if the issue rise again.
If it's working fine please feel free to close this case.

**Comment by JohnnyTex** [comment]
Unfortunately it's not working fine. I can very safely say this time that this driver initially worked fine, but as the day went on and I tried to load up the game for another session I had the same issue as before. I haven't heard back from anyone on the Discord, but it's basically a mirror situation of what was going on in 4091.

**Comment by IntelSupport-Arun** [comment]
Hi @JohnnyTex ,

Since issue is happening again I would like to know below details:
1. How many displays are you using?  All displays connected to ARC?
2.  Is issue occurred while launching the game or while playing the game after long hours of playing?
3.  Issue is intermittent or frequent? What is the failure rate?
4.  Any crash log has been generated? if yes, please attach
5.  Any screenshot or video of the issue that you can provide for our reference

**Comment by JohnnyTex** [comment]
1. For me it will always be one, I don't use multiple displays. 
2. When launching the game. 
3. Frequent, since I've reported I've been totally unable to play this game. 
4. I sadly don't think this game generates crash logs. 
5. The error code in particular is 0xC0000005 N

**Comment by JohnnyTex** [comment]
I may have found the error; when I disabled my CPU's integrated graphics the game began to work again. That might be the culprit.

**Comment by JohnnyTex** [comment]
Ever since disabling the Integrated Graphics, this game has been running w/o issue on driver 4123. I consider the issue closed.

**Comment by kunit1** [comment]
This issue shouldn't be closed as it's not normal to expect the integrated graphics to be disabled to play the game. Clearly the game works with both Nvida and AMD with intel integrated graphics enabled. Intel needs to fix the regression in their drivers.

**Comment by JohnnyTex** [comment]
Fair point. I'll reopen it and wait for an official response.

**Comment by IntelSupport-Arun** [comment]
Hi @JohnnyTex ,

There is a new Beta driver version [31.0.101.4125](https://www.intel.com/content/www/us/en/download/729157/intel-arc-iris-xe-graphics-beta-windows.html) has been released. Could you check and confirm the issue is still seeing with the latest driver?

**Comment by JohnnyTex** [comment]
I'll look into it when I can this week.

**Comment by IntelSupport-Arun** [comment]
Hi @JohnnyTex ,
I have verified the issue on Core i7 12700K  with [31.0.101.4125](https://www.intel.com/content/www/us/en/download/729157/intel-arc-iris-xe-graphics-beta-windows.html) and I couldnt able to reproduce the issue. Im able to play the game and no issue observed.
Are you still seeing the issue? Please confirm.

**Comment by JohnnyTex** [comment]
Sorry, university has been roughing me up as of late, I'll try and get it on it soon. I'll note thought that typically speaking it works fine at first for me, as long as a day or so, but once it happens there's no way of going back.

**Comment by JohnnyTex** [comment]
Testing on 4146, I can safely say the game still crashes w/ iGPU enabled.

**Comment by Arturo-Intel** [comment]
Hello @JohnnyTex 
I was unable to reproduce this issue on the 4255 driver version

Can you verify this on your side? remember to use DDU for a clean reinstallation.

**Comment by JohnnyTex** [comment]
I'm on 4257 and it actually happened to me just yesterday or so. Though the error code is different now. Having iGPU on also sporadically messed with other things like video corruption when turning on the telemetry overlay and whatnot in otherwise compatible games like Valorant. Next time I boot the game up I'll be sure to give it another shot.

**Comment by Arturo-Intel** [comment]
<img width="364" alt="image" src="https://user-images.githubusercontent.com/92489452/229588825-9739e591-4214-45fd-a6d4-d09813d5bf1b.png">

This is my setting

**Comment by JohnnyTex** [comment]
Just checked and yup, it's set to high performance. In fact, all my games are manually set to high.

**Comment by JohnnyTex** [comment]
Also, I swore it had given me a different error message, but it turns out it was the one I already reported.

**Comment by Arturo-Intel** [comment]
Its really strange, I am trying and trying to get that error. Do you have any other Sw running, like OBS, or some streamer tools, something like that?

**Comment by JohnnyTex** [comment]
No, I never run sw like that (not a content creator). This is definitely concerning, bc I can replicate this error on command. Is there any way this could be a 12600k-specific error? Unlikely, but a possibility perhaps.

**Comment by Arturo-Intel** [comment]
Hmm I dont think it is related to the 12th gen cpu, I have one 12900K and cant reproduce the issue, the other thing maybe could be a game corruption, did you try to reinstall it before?

**Comment by JohnnyTex** [comment]
Yes, many times. I'm not exactly excited at the idea of having to do that again, but I can if there are no other explanations. The issue ofc is that the error code the game gives me faults the graphics hardware.

**Comment by Arturo-Intel** [comment]
Also try to verify the game files integrity:
"For Battle.net Launcher, launch the Battle.net client. Tap the cogwheel-shaped icon that will be next to the Play button. Now, select Scan & Repair from the drop-down section. Hit Begin Scan and wait."

**Comment by JohnnyTex** [comment]
I'm pretty sure I run the game in admin mode. I can check soon. I scanned game files many, many times. There was never an issue. I can look up what the COD Support page says the error code is when I run into it again. I'm not sure exactly when I can check though, as I'm preparing for finals/graduation.

**Comment by Arturo-Intel** [comment]
Well, I can play perfectly the game with the iGPU enabled from Battle.net .
I am out of ideas...

**Comment by JohnnyTex** [comment]
I currently can't test this due to my PC having ram issues, so I'll give it another try later.

**Comment by JohnnyTex** [comment]
I think I may have actually been targeting the wrong thing perhaps; it might not necessarily be the iGPU that's to blame, but perhaps it's the iGPU multi-monitor implementation that's to blame. I don't have a dual-monitor setup, but I had the option in the BIOS turned on (the one that says it uses the iGPU in the event of multimonitor usage). When I disable that option, all my troubles are gone. But was that actually me turning off the iGPU? Or just a function of it?

**Comment by Arturo-Intel** [comment]
Hmm, let me do a quick check on my side, maybe that's why I cant reproduce it

**Comment by Arturo-Intel** [comment]
Yup, I have it enabled 
![image](https://user-images.githubusercontent.com/92489452/236517708-f251e437-096d-415d-94e5-627df8685500.png)
And I can play the game with no issue
BTW I am using 31.0.101.4314

**Comment by Arturo-Intel** [comment]
Are you using HDMI or Display-port?

**Comment by Arturo-Intel** [comment]
Hey @JohnnyTex:
I was having the same behavior you describe (disabling iGPU or the multi monitor option was the only way to play the game, if not the game didnt get to the main menu screen) after a ugly error message pops up when i was trying to play the game to do some other tests. 
After several tries and debug, I finally desist and reinstall the game... with the same result u_u
So I try to reinstall the Intel drivers (using DDU) and the behavior was no more.
The only difference is that I installed another version (and older one) 
I will install the same version I had before (31.0.101.4314) and share the results

--r2

**Comment by Arturo-Intel** [comment]
Yup, that will do it
@JohnnyTex please do a DDU safemode reboot and reinstall the drivers

--r2

**Comment by JohnnyTex** [comment]
1) I'm using DP
2) Well, I could ostensibly do that myself, but I always DDU'd my drivers back then. I think the safe mode reboot is probably the move, cause I've never used safe mode before.

**Comment by Arturo-Intel** [comment]
@JohnnyTex any luck? does the save mode helped?

**Comment by JohnnyTex** [comment]
Haven't tried it yet, give me the week and I'm sure I'll have the opportunity to.

**Comment by JohnnyTex** [comment]
My bad, i forgot to update here that my monitor is currently under warranty repair, I'll hop on this the moment it comes back.

**Comment by JohnnyTex** [comment]
I kid you not, the safe mode might have done the trick. I'll give it a couple days though due to the degradation trait I observed earlier, but so far it looks like safe mode was the move. I'll update this in a couple days to confirm if he fix works for real.

**Comment by JohnnyTex** [comment]
Alright, it's been 3 days and every time I've booted the game up it's worked w/ iGPU turned on. Safe mode DDU + Clean Install did the trick.
